src/components/backend-ai-agent-list.ts

```diff
@@ -182,18 +196,18 @@ export default class BackendAIAgentList extends BackendAIPage {
     switch (this.condition) {
     case 'running':
       status = 'ALIVE';
-      fields = ['id', 'status', 'version', 'addr', 'region', 'compute_plugins', 'first_contact',
+      fields = ['id', 'status', 'version', 'addr', 'region', 'compute_plugins', 'first_contact', 'schedulable',
```

Comment:
`schedulable` mutation exists only for Manager>=21.09 so this query will raise an error on Manager<=21.03. Since Web-UI main branch currently supports multiple versions of Managers, I think the `schedulable` field should be version-checked. Version checking might be performed in the JS SDK library, `agent.list`.